script/check-size

```diff
@@ -126,6 +126,7 @@ size_nrf52840()
         "SNTP_CLIENT=1"
         "TIME_SYNC=1"
         "UDP_FORWARD=1"
+        "THREAD_VERSION=1.2"
```

Comment:
The size report is not showing any differences. Is that expected?

Reply:
I misunderstood how `check-size` works. 
Turn out with this PR, it's comparing 1.2 with 1.2, so no change at all. 
Maybe we should close this PR since it's useless.